### ISSUE TITLE
Twintris by Twinsen (POM '98) (PD) causes core error

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#132]
Originally reported on Google Code with ID 132

```
ROM: Twintris by Twinsen (POM '98)
MD5: A74738D03ACA4E51690408D203E86D49

Happens with all 3 video plugins. Give error:

unknown task:
        type:0
        sum:214bf
        PC:8e0567c


```

Reported by `sknauert@wesleyan.edu` on 2008-07-12 04:11:29

**Comment by richard42** [comment]
```
I'm pretty sure this is related to the rsp-hle plugin
```

Reported by `jezter82` on 2008-12-23 06:18:53

**Comment by richard42** [comment]
```
Just as update: Works with Z64gl LLE with Z64 RSP-LLE
```

Reported by `sven@narfation.org` on 2009-10-08 19:07:40

**Comment by richard42** [comment]
```
This error was caused by RSP HLE plugin not forwarding GFX ucode to video plugin.
This has been fixed by commit 1e306f0.

```

Reported by `bobby.smiles32` on 2013-03-31 22:35:23

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2013-12-18 10:52:21
- **Status changed**: `Fixed`
